cookbooks/Getting-Started/eval/promptfoo/travel.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import os
+
+import openai
+from dotenv import load_dotenv
+
+from aiconfig import AIConfigRuntime
+from aiconfig.model_parser import InferenceOptions
+import sys
+
+
+async def main():
```

Comment:
This script seems like something that should be an interface or function in the core library which can set this up.

Inputs to it: the aiconfig, and prompt to run. Rest is boilerplate .

Or even more general, could take in an AIConfigRuntime object and a run function

Reply:
Yeah makes sense, almost all the boilerplate here (probably?) can be automated. There doesn't seem to be a great way to run it as a library though - the most sensible procedure I can think of is to have an AIConfig testing util script. Something like:

```
python aiconfig_test.py setup path/to/aiconfig prompt_name path/to/new/config.yaml
python aiconfig_test.py test path/to/new/config.yaml
```

Reply:
Actually you don't even need the `test` subcommand. It's just `npx promptfoo@latest eval`.

Reply:
> Or even more general, could take in an AIConfigRuntime object and a run function

At that point, why not have the user write their own script? I think inverting control at that level of complexity can make debugging harder more than it makes authoring easier.